src/jumio-api/interfaces/jumio-transaction-retrieve.ts

```diff
@@ -192,7 +226,7 @@ export interface JumioTransactionRetrieveResponse {
       id: string;
       credentials: [
         {
```

Comment:
There were 1. incorrect type value, 2. undocumented undefined fields in the API response